packages/react-aria-components/docs/Calendar.mdx

```diff
@@ -880,14 +914,12 @@ function WeekCalendarGrid({state, ...props}: WeekCalendarGridProps) {
 
 ```tsx example
 <Calendar visibleDuration={{weeks: 1}} defaultValue={today(getLocalTimeZone())}>
-  {({state}) => (
-    <div className="week">
-      <Heading />
-      <Button slot="previous">◀</Button>
-      <WeekCalendarGrid state={state} />
-      <Button slot="next">▶</Button>
-    </div>
-  )}
```

Comment:
Now that we can access the component state via a public context, what do we think about the existence of state in renderProps? I guess it can come in handy if the user wants to style something/do something simple without needing to make a wrapper component and access state via the state context, but feels like it is largely unneeded now.

Reply:
I guess there's no reason not to expose it. Like you said it could be useful for one-offs that don't need to be a whole new reusable component.